pkgs/top-level/all-packages.nix

```diff
@@ -6281,6 +6281,8 @@ in
 
   cgdb = callPackage ../development/tools/misc/cgdb { };
 
+  cheat = callPackage ../data/documentation/cheat { };
```

Comment:
`application/misc` is fine for me. Though, I should note that `application/office` doesn't mean application should deal with large nicely formatted documents and this kind of things.

In example, we currently have [todo.txt-cli](http://todotxt.com/) and [tudu](https://code.meskio.net/tudu/) in `application/office`, which both are small apps for managing todo lists. There are also [keepnote](http://keepnote.org/) and [eventlist](https://store.kde.org/content/show.php/Eventlist?content=107779), which are GUI apps for organizing other things.